### ISSUE TITLE
Presets -- not showing in the toggle down menu after adding to user_presets

### ISSUE THREAD

**Issue by rafstahelin** [bmaltais/kohya_ss#2551]
Is there a special way to add your custom preset in "user_presets"

I have added my own json file inside the Lora > user_presets folder but it does not show up in the toggle menu. 

I have checked it is formatted as a json. In fact i saved it from config save button so it should pan out. 

i have restarted the process.

Still nothing, not sure what else it could be

**Comment by b-fission** [comment]
After restarting the gui and reloading the webpage, it should appear at the bottom of the list.

I copied the json file directly into user_presets without any editing.

![preset](https://github.com/bmaltais/kohya_ss/assets/131207849/cd58b1a3-c5c4-437d-bad4-fdcb5d8ccaa4)

**Comment by rafstahelin** [comment]
thanks it did work. for some reasons my json files didnt have extension when i downloaded from runpod. seems to be a weird glitch. now showing fine on preset toggle. thanks again

**Comment by rafstahelin** [comment]
:-)

**Comment by rafstahelin** [comment]
@b-fission just a weird question. My usual Learning rate for Lora module SDXL adamw8bit training was always 1e-4 with xformers. I recently started using sdpa. Worked fine in Finetune module. But in lora, its outputting garbled samples. I am setting TE learning at 1/10th the lr of unet. 

Am i missing something? 

[lora-adamw8bit-sdpa-1e-4.json](https://github.com/user-attachments/files/15521741/lora-adamw8bit-sdpa-1e-4.json)
